website/server/controllers/api-v3/shops.js

```diff
@@ -0,0 +1,122 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import shops from '../../../../common/script/libs/shops';
+
+let api = {};
+
+/**
+ * @apiIgnore
+ * @api {get} /api/v3/shops/market get the available items for the market
+ * @apiVersion 3.0.0
+ * @apiName GetMarketItems
+ * @apiGroup Shops
+ *
+ * @apiSuccess {Object} data List of push devices
+ * @apiSuccess {string} message Success message
+ */
+api.getMarketItems = {
+  method: 'GET',
+  url: '/shops/market',
+  middlewares: [authWithHeaders()],
+  async handler (req, res) {
+    let user = res.locals.user;
+
+    let resObject = {
+      identifier: 'market',
+      text: res.t('market'),
+      notes: res.t('welcomeMarketMobile'),
```

Comment:
different capabilities of the markets. mobile doesn't have selling at that place and also doesn't offer some special items at the moment